plugins/inputs/beat/beat_test.go

```diff
@@ -0,0 +1,225 @@
+package beat
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+var beatTest = NewBeat()
```

Comment:
Please move this to the individual tests.

Reply:
will fix